src/View/layout.html.twig

```diff
@@ -6,11 +6,13 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="icon" href="/assets/images/favicon.png">
 		<title>
-			{% block title %}{% endblock %}
+			{% block title %}H&G{% endblock %}
 		</title>
-		<link rel="stylesheet" href="/assets/css/style.css">
+					
 		{% block stylesheet %}
-			<link rel="stylesheet" href="/assets/css/header.css">
+			<link rel="stylesheet" type="text/css" href="/assets/css/style.css">
```

Comment:
This file must be outside the stylesheet block since it will be available on all the website